src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -661,20 +659,11 @@ private static void NoExitPoll()
     }
     internal static void MyDebugBreak(string extraData)
     {
-#if !PROJECTK_BUILD
```

Comment:
Do we still want to keep the variant where `PROJECTK_BUILD` is not defined?
It looks like the code won't compile (with or without your change) without `PROJECTK_BUILD` defined.

Reply:
yeah, I thought about just getting rid of the define (ie, it's always defined) since I don't see any value of maintaining it. but didn't do it with this PR... feel free to do that if you like.